deployments/charts/kuma/templates/cp-global-sync-service.yaml

```diff
@@ -2,17 +2,19 @@
 apiVersion: v1
 kind: Service
 metadata:
-  name: {{ include "kuma.name" . }}-global-remote-sync
+  name: {{ include "kuma.controlPlane.globalRemoteSync.serviceName" . }}
   namespace: {{ .Release.Namespace }}
+  annotations:
+    {{- range $key, $value := .Values.controlPlane.globalRemoteSyncService.annotations }}
+      {{ $key }}: {{ $value | quote }}
+    {{- end }}
 spec:
-  {{- if .Values.controlPlane.useNodePort }}
-  type: NodePort
-  {{- else }}
-  type: LoadBalancer
-  {{- end }}
+  type: {{ .Values.controlPlane.globalRemoteSyncService.type }}
   ports:
     - port: 5685
```

Comment:
Shall we parametrize this port too? Just in can and while we are here.

Reply:
done